modules/lily-compiler/src/main/java/io/github/tomboyo/lily/compiler/cg/AstClassCodeGen.java

```diff
@@ -70,4 +73,13 @@ private static String recordField(Field field) {
           scope);
     }
   }
+
+  private static String implementsClause(AstClass astClass) {
+    if (astClass.interfaces().isEmpty()) {
+      return "";
+    }
+
+    return "implements "
+        + astClass.interfaces().stream().map(Fqn::toFqpString).collect(Collectors.joining(", "));
+  }
```

Comment:
Is it worth putting this function somewhere else since its defined exactly the same twice?